src/forms/labels/labeled-field.js

```diff
@@ -72,10 +73,12 @@ const propTypes = {
   ...InputError.propTypes,
   children: PropTypes.node,
   hideErrorLabel: PropTypes.bool,
+  as: PropTypes.string,
```

Comment:
Do we want to be this permissive or are `div` and `fieldset` the only viable options?

Reply:
I think we should be strict with what's allowed and if we ever receive requests re: expanding the options (for whatever reason that I can't think of atm), we can address then!

Reply:
cc: @danparnella